Description: config: fix power8 default conf

Latest dpdk FTBFS for power8 default config.
This (re-)enables a config symbol as recommended by IBM
(gowrishankar.m@in.ibm.com)

Signed-off-by: Christian Ehrhardt <christian.ehrhardt@canonical.com>

Origin: No, will be pushed by IBM
Author: Christian Ehrhardt <christian.ehrhardt@canonical.com>
Last-Update: 2016-09-02
--- a/config/defconfig_ppc_64-power8-linuxapp-gcc
+++ b/config/defconfig_ppc_64-power8-linuxapp-gcc
@@ -51,7 +51,6 @@
 CONFIG_RTE_LIBRTE_IXGBE_PMD=n
 CONFIG_RTE_LIBRTE_VIRTIO_PMD=y
 CONFIG_RTE_LIBRTE_VMXNET3_PMD=n
-CONFIG_RTE_LIBRTE_PMD_BOND=n
 CONFIG_RTE_LIBRTE_ENIC_PMD=n
 CONFIG_RTE_LIBRTE_FM10K_PMD=n
 CONFIG_RTE_LIBRTE_SFC_EFX_PMD=n
